neps/api.py

```diff
@@ -311,6 +317,9 @@ def _run_args(
         # Support pipeline space as ConfigurationSpace definition
         if isinstance(pipeline_space, CS.ConfigurationSpace):
             pipeline_space = pipeline_space_from_configspace(pipeline_space)
+        # Support pipeline space as YAML file
+        elif isinstance(pipeline_space, str):
+            pipeline_space = pipeline_space_from_yaml(pipeline_space)
```

Comment:
Although `Path` is also supported as a type to be passed in the `pipeline_space` argument. There is no condition for it here and how it should be handled internally.

Currently having a `Path` type pipeline_space could result in an error because it will skip these checks.

Reply:
The usage of the Path object from the pathlib module is now supported [here](https://github.com/automl/neps/pull/46/commits/c70b85ee5c65ed178b84e7c11e94ed8110e2b6ab)